static/app/components/profiling/flamegraph.tsx

```diff
@@ -90,15 +254,28 @@ function Flamegraph(props: FlamegraphProps): ReactElement {
 
       <FlamegraphZoomViewMinimapContainer height={flamegraphTheme.SIZES.MINIMAP_HEIGHT}>
         <FlamegraphZoomViewMinimap
-          flamegraph={flamegraph}
+          canvas={flamegraphMiniMapCanvas}
```

Comment:
I would rename `canvas` prop to `flamegraphCanvas` so that it doesnt imply a canvas HTML element